tenant/service_onboarding.go

```diff
@@ -61,6 +61,10 @@ func (s *OnboardService) onboardUser(ctx context.Context, req *influxdb.Onboardi
 		return nil, ErrOnboardInvalid
 	}
 
+	if len(req.Password) < 8 {
```

Comment:
Could this be moved inside the `if req.Password != "" {` There is a circumstance we would want to onboard the a user and not provide a password for them.